api/src/wfl/executor.clj

```diff
@@ -285,15 +292,37 @@
            (jdbc/query tx)
            first))))
 
+(defn ^:private method-content-url
+  "Return the URL of the WDL contents associated with this `method`."
+  [{:keys [methodPath methodVersion] :as method}]
+  (throw-unless-dockstore-method method)
+  (let [id  (url-encode (str "#workflow/" methodPath))
+        url (:url (dockstore/ga4gh-tool-descriptor id methodVersion "WDL"))]
+    (if-let [groups (re-find #"^https://raw.githubusercontent.com/(.*)$" url)]
+      ;; use jsdelivr to avoid GitHub rate-limits
+      (let [[author repo version path] (str/split (second groups) #"/" 4)]
+        (str/join "/" ["https://cdn.jsdelivr.net/gh" author (str repo "@" version) path]))
+      url)))
```

Comment:
Move `;;` comment on top of `defn`?

Reply:
why? It describes why I'm doing this, it's not relevant to the workings of the whole function

Reply:
moved into docstring